e3nn/nn/models/v2103/voxel_convolution.py

```diff
@@ -60,7 +60,12 @@ def __init__(self, irreps_in, irreps_out, irreps_sh, size, steps=(1, 1, 1), **kw
         sh = o3.spherical_harmonics(self.irreps_sh, lattice, True, 'component')  # [x, y, z, irreps_sh.dim]
         self.register_buffer('sh', sh)
 
-        self.tp = FullyConnectedTensorProduct(self.irreps_in, self.irreps_sh, self.irreps_out, shared_weights=False)
+        self.tp = FullyConnectedTensorProduct(
+            self.irreps_in, self.irreps_sh, self.irreps_out,
+            shared_weights=False,
+            compile_left_right=False,
```

Comment:
Can you explain the name of this flag? Why `left_right`? Does it compile both `forward` and `right`? In which case, why isn't there a way to only compile `forward`?

Reply:
left_right means that you tensor product with both left and right hand side. 

forward == left_right
.right == right

Reply:
👍